shared-libs/contacts/test/unit/places.spec.js

```diff
@@ -336,24 +336,66 @@ describe('places controller', () => {
           type: 'person'
         }
       };
+      
       sinon.stub(people, 'getOrCreatePerson').resolves({
         _id: 'qwe',
+        _rev: '1',
         name: 'Jim',
         type: 'person'
       });
-      fetchHydratedDoc.resolves({
-        _id: 'ad06d137',
-        name: 'CHP Branch One',
-        type: 'district_hospital'
+      db.medic.post.withArgs(
+        sinon.match(
+          (obj) => {
+            // place is saved first, then updated with contact later
+            return obj.name === 'CHP Family' && !obj.contact;
```

Comment:
Nit: testing that the name is `'CHP Family'` is redundant here, especially since you assert it in the callback. To make this more compact and less of a read, you can change to: 

```
   db.medic.post
     .withArgs(sinon.match((obj) => !obj.contact))
     .callsFake(doc => {
        chai.expect(doc.name).to.equal('CHP Family');
        chai.expect(doc.parent._id).to.equal('ad06d137');
        return Promise.resolve({ id: 'hc', rev: '1' });
      });
```